fix_appdata_file.patch

```diff
@@ -20,7 +20,7 @@ diff -ruN a/misc/qelectrotech.appdata.xml b/misc/qelectrotech.appdata.xml
   <summary xml:lang="ja">電気回路図エディタ</summary>
 + <content_rating type="oars-1.1"/>
 + <releases>
-+     <release version="0.8.0" date="2021-02-21">
++     <release version="0.8.0" date="2021-02-21"/>
```

Comment:
You're still missing the `</releases>` end tag.

Reply:
No, this is xml and it's correct.